test/Conversion/TorchToLinalg/basic.mlir

```diff
@@ -67,3 +67,17 @@ func @integer_extract(%arg0: !torch.vtensor<[],si64>) -> !torch.int {
   %0 = torch.aten.Int.Tensor %arg0 : !torch.vtensor<[],si64> -> !torch.int                   
   return %0 : !torch.int                 
 }                       
+
+// -----
+
+// CHECK:    func @ntt(%[[IN:.*]]: !torch.int) -> !torch.vtensor<[],si64> {
+// CHECK:      %[[A:.*]] = torch_c.to_i64 %[[IN]]
+// CHECK:      %[[B:.*]] = linalg.init_tensor [] : tensor<i64>
+// CHECK:      %[[C:.*]] = linalg.fill(%[[A]], %[[B]]) : i64, tensor<i64> -> tensor<i64> 
+// CHECK:      %[[OUT:.*]] = torch_c.from_builtin_tensor %[[C]] : tensor<i64> -> !torch.vtensor<[],si64>
+// CHECK:      return %[[OUT]] : !torch.vtensor<[],si64>
+
+func @ntt(%arg0: !torch.int) -> !torch.vtensor<[],si64> {
```

Comment:
sry that I missed this in the previous review. It's better to name the function `torch.prim.NumToTensor.Scalar`. `ntt` is not a common acronym.